code/products/mysql/connect-using-mysqlx.py

```diff
@@ -0,0 +1,26 @@
+import mysqlx
+
+MYSQLX_PASSWORD = "user-password"
```

Comment:
Because we explain variables in Variable section, I think duplicating is not necessary these three parameters at the start. It doesn't work as part of the code, right?

Reply:
this would not be a valid code. However, I added a comment so the user can replace it with their variables.